db/seeds.rb

```diff
@@ -1,7 +1,128 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+# This file should contain all the record creation needed to seed the database
```

Comment:
Agree with Dennis - for staff tracker rubocop we exclude all files in the db folder.